src/__tests__/__snapshots__/dev.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`can be configured to use the displayName in the className 1`] = `
+.glamor-0,
+[data-glamor-0] {
+  label: Hi There;
```

Comment:
I want to hold off on merging this until this is fixed. How can I help @threepointone?